sdk/eventhub/Azure.Messaging.EventHubs/tests/Amqp/AmqpConsumerTests.cs

```diff
@@ -499,5 +500,32 @@ public async Task ReceiveAsyncValidatesClosed()
 
             Assert.That(async () => await consumer.ReceiveAsync(100, null, cancellationSource.Token), Throws.InstanceOf<EventHubsException>().And.Property(nameof(EventHubsException.Reason)).EqualTo(EventHubsException.FailureReason.ClientClosed));
         }
+
+        /// <summary>
+        ///   Verifies functionality of the <see cref="AmqpConsumer.ReceiveAsync" />
+        ///   method.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ReceiveAsyncValidatesConnectionClosed()
+        {
+            var endpoint = new Uri("amqps://not.real.com");
+            var eventHub = "eventHubName";
+            var consumerGroup = "$DEFAULT";
+            var partition = "3";
+            var eventPosition = EventPosition.FromOffset(123);
+            var options = new EventHubConsumerClientOptions();
+            var retryPolicy = new BasicRetryPolicy(new EventHubsRetryOptions());
+            var mockCredential = new EventHubTokenCredential(Mock.Of<TokenCredential>());
+
+            var scope = new AmqpConnectionScope(endpoint, endpoint, eventHub, mockCredential, EventHubsTransportType.AmqpTcp, null);
+            scope.Dispose();
+
+            var consumer = new AmqpConsumer(eventHub, consumerGroup, partition, eventPosition, true, null, null, null, scope, Mock.Of<AmqpMessageConverter>(), retryPolicy);
+            await consumer.CloseAsync(CancellationToken.None);
+
+            Assert.That(async () => await consumer.ReceiveAsync(100, null, CancellationToken.None),
```

Comment:
Wouldn't this test have passed before the fix since we are explicitly closing the consumer?

Reply:
d'oh... yeah.   L525 is left over from when I cloned the test and should have been removed.